### DISCUSSION TITLE
附录显示问题

### DISCUSSION THREAD

**Discussion by Yellowwwwwww121** [note286/xduts#51] [Q&A] [answered]
问题描述：添加附录章时，附录内容接着第五章显示了，没有单独变成附录，日志没有报错。
代码：info/appendix = {information/appendix.tex}, %开头添加
 （我只在开头添加了附录文件的路径，document环境中没添加附录）         
错误形式如下：
![image](https://user-images.githubusercontent.com/126239297/221200736-8f63f731-d17c-46c8-af2c-20b2a4e39264.png)

**Comment by note286** [comment, marked as answer]
`appendix.tex`内请使用`\chapter{}`。

**Comment by Yellowwwwwww121** [comment]
感谢感谢，问题解决
